gargoyle/README.md

```diff
@@ -18,3 +18,11 @@ haskellPackages.override {
     };
 }
 ```
+
+# Hacking
+
+Do something like this:
+
+```shell
+nix-shell -A gargoyle-postgresql.env --run 'cd gargoyle-postgresql && cabal new-repl'
```

Comment:
Personally, I've gotten in the habit of adding top-level `repl`, `watch`, `hoogle` (and `run` for executables) scripts with the appropriate commands. It's harder to forget to update those because your workflow immediately breaks, and once you fix it, you'll want to commit it so git stops bothering you about those changes.

Reply:
That's reasonable. I would consider that independent of these changes, no? You could submit a PR for that.

Reply:
Oh sure